### PR TITLE
Support to build binaries for `ppc64le` architecture.

### PR DESCRIPTION
**Pull request by kishen-v**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
This PR has changes to build binaries for the `ppc64le` CPU architecture and the related user documentation changes.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test

thanks @kishen-v !

**Comment by haircommander** [comment]
the test failures are unrelated. This could be rebased to pickup some fixes, but I don't think it's necessary
/approve

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -166,12 +166,35 @@ jobs:
...
+  build-static-ppc64le:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v20
+        with:
+          install_url: https://releases.nixos.org/nix/nix-${{ env.NIX_VERSION }}/install
+      - uses: cachix/cachix-action@v12
+        with:
+          name: cri-o-static
```
Comment (saschagrunert) [inline .github/workflows/test.yml:178]: The cache should have a ppc64le suffix, otherwise we will override it.

**Review by saschagrunert — COMMENTED** [review]
Just one nit about the cache, otherwise great work! Thank you for the contribution!

File: .github/workflows/test.yml (hunk 1 above)
Comment (kishen-v) [reply .github/workflows/test.yml]: Hi @saschagrunert,
Renamed `cri-o-static` to `cri-o-static-ppc64le` as suggested.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -166,12 +166,35 @@ jobs:
...
+  build-static-ppc64le:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: cachix/install-nix-action@v20
+        with:
+          install_url: https://releases.nixos.org/nix/nix-${{ env.NIX_VERSION }}/install
+      - uses: cachix/cachix-action@v12
+        with:
+          name: cri-o-static-ppc64le
```
Comment (saschagrunert) [inline .github/workflows/test.yml:178]: Ah my fault, let's revert back to the existing cachix cache:
```suggestion
          name: cri-o-static
```

Comment (kishen-v) [reply .github/workflows/test.yml]: Hi @saschagrunert,
Reverted the changes as suggested.

**Review by saschagrunert — APPROVED** [review]

**Comment by kishen-v** [comment]
The `test / bundles (pull_request)` job fails during step`make bundles` due to non-availability of conmon for ppc64le.
https://github.com/cri-o/cri-o/actions/runs/4933971318/jobs/8818878101. I will add the required changes.  

Thanks!

**Pull request merged**